tests/subsys/fs/multi-fs/testcase.yaml

```diff
@@ -2,7 +2,3 @@ tests:
   filesystem.multifs:
     platform_whitelist: qemu_x86
     tags: nffs fatfs filesystem
-  filesystem.fs_shell:
-    extra_args: CONF_FILE="prj_fs_shell.conf"
```

Comment:
Both tests of fs_shell & multifs use qemu_x86 as platform whitelist, how come only "prj_fs_shell.conf" is causing issues and not prj.conf?

I am also referring below comment:
I am going to remove tests/subsys/fs/multi-fs/prj_fs_shell.conf because this project can be only build for qemu and qemu does not have DT_FLASH_DEV_NAME.